ui/src/overridden/backoffice/utils.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { config } from "../../config";
+
+export const formatPrice = (price, includeCurrency = true) => {
```

Comment:
Why not re-using the already defined func [here](https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/api/utils.js#L69)?